rosa_install_access_delete_clusters/terraform/rosa-classic-creating-a-cluster-quickly-terraform.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="rosa-classic-creating-a-cluster-quickly-terraform"]
+= Creating a default ROSA (classic architecture) cluster using Terraform
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-classic-creating-a-cluster-quickly-terraform
+
+toc::[]
+
+Quickly create a {product-title} (ROSA) cluster quickly by using a Terraform cluster template that is configured with the default cluster options.
+
+The cluster creation process described below uses a Terraform configuration that prepares a ROSA Classic AWS Security Token Service (STS) cluster with the following resources:
```

Comment:
```suggestion
The cluster creation process described below uses a Terraform configuration that prepares a ROSA (classic architecture) AWS Security Token Service (STS) cluster with the following resources:
```
Similarly, you may want to ask about how often we should be repeating "(classic architecture)". Is it only part of the first mention? Something we do every time we write out ROSA? We will need clarification for how we are handling it now (when both the ROSA (classic architecture) and ROSA docs are together) and if we want to handle it differently when we have two different sets of docs in the future. Right now, we need every page to mention "ROSA (classic architecture)" at least once so that our users know they are in the correct section for their specific product.